docs/metalk8s-ui.rst

```diff
@@ -0,0 +1,49 @@
+Metalk8s UI
+===========
+
+
```

Comment:
minor: why so many empty lines here: 1 or 2 should be enough

Reply:
It was for a better visibility, I deleted some line.
Effective with the next commit.
THX